README.md

```diff
@@ -7,18 +7,9 @@
 * Easy to extend to fit any backend or additional frontend tools. 
 
 ## Get Started
+To get started just `pip install -r requirements.txt` and `make run`.
```

Comment:
The html rendering doesn't recognize code block is weird so I changed this back.